### PR TITLE
Talbpaul/issue mr templates

### LINKED ISSUE #3: New issue and merge request templates
Our old gitlab merge request and issue request templates need to be revisited now that the repository is hosted in github.

Our labels aren't set up yet and I can't see where to add them, but this is an Improvement with Normal priority.

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
--------
Pull Requeset Description
--------
##### What issue does this change request address?
This will close #3.

##### What are the significant changes in functionality due to this change request?
Adds/updates the issue/change request templates to the new repository.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://hpcgitlab.inl.gov/idaholab/raven/wikis/Developer_Information#python-code-standard) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+##### What are the significant changes in functionality due to this change request?
+
+
+----------------
+For Change Control Board: Change Request Review
+----------------
+The following review must be completed by an authorized member of the Change Control Board.
+- [ ] 1. Review all computer code.
+- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
+- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://hpcgitlab.inl.gov/idaholab/raven/wikis/Developer_Information#python-code-standard) for details.
```
Comment (joshua-cogliati-inl) [inline .github/PULL_REQUEST_TEMPLATE.md]: We need to copy the standard to the wiki, or reference the Test Plan document instead.

**Comment by joshua-cogliati-inl** [comment]
I have reviewed the changes and had one comment.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (PaulTalbot-INL) [inline .github/PULL_REQUEST_TEMPLATE.md]: Ha, yes, this is true.  Do you have a preference?  I like the wiki myself.  I can do that if you want.

Comment (joshua-cogliati-inl) [inline .github/PULL_REQUEST_TEMPLATE.md]: Sure, feel free to make the change.

**Comment by PaulTalbot-INL** [comment]
I made it a standalone page, since it contains quite a bit more information than when we first put it in "developer information".  In the future as we construct the contribution guide, we can link to it from there.

**Comment by joshua-cogliati-inl** [comment]
I have reviewed the changes.  There are no code or build changes, so testing is not required.

**Pull request merged**